modules/nodes-nodes-garbage-collection-configuring.adoc

```diff
@@ -5,55 +5,65 @@
 [id='nodes-nodes-garbage-collection-configuring_{context}']
 = Configuring garbage collection for containers and images
 
-As an administrator, you can configure how {product-title} performs garbage collection for each type of node group.
+As an administrator, you can configure how {product-title} performs garbage collection.
 
-.Procedure
-
-. Open the appropriate node configuration map. 
+.Prerequisites 
+	
+. Obtain the label associated with the static Machine Config Pool CRD for the type of node you want to configure. 
+	Perform one of the following steps:
+	
+.. View the Machine Config Pool:
 +
-* node-config-master
-
-* node-config-infra
-
-* node-config-compute
-
-* node-config-all-in-one
-
-* node-config-master-infra
-
-
-. For containers, you can specify values for these settings in the `*kubeletArguments*` section of
-the node configuration map. Add the section if it does not already exist:
+[source,bash]
+----
+# oc describe machineconfigpool <name>
+----
++
+For example:
 +
 [source,yaml]
 ----
-kubeletArguments:
-  minimum-container-ttl-duration: <1>
-    - "10s"
-  maximum-dead-containers-per-container: <2>
-    - "2"
-  maximum-dead-containers: <3>
-    - "240"
+# oc describe machineconfigpool worker
+
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  creationTimestamp: 2019-02-08T14:52:39Z
+  generation: 1
+  labels:
+    custom-kubelet: small-pods <1>
 ----
-<1> Specify the minimum age that a container is eligible for garbage collection. The
-default is *0*. 
-<2> Specify the number of instances to retain per pod container. The default is *1*.
-<3> Specify the maximum number of total dead containers in the node. The default is *-1*, which means unlimited.
+<1> If a label has been added it appears under `labels`.
 
-. For images, you can specify values for these settings in the `*kubeletArguments*` section of
-the node configuration map.
+.. If the label is not present, add a key/value pair:
 +
+[source,bash]
+----
+oc label machineconfigpool worker custom-kubelet=small-pods
+---- 
+
+.Procedure
+
+. Create a CRD for your configuration change.
++
+.Sample configuration for a *garbage* CRD
 [source,yaml]
 ----
-kubeletArguments:
-  image-gc-high-threshold: <1>
-    - "85"
-  image-gc-low-threshold: <2>
-    - "80"
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: set-garbage-collection <1>
+spec:
+  machineConfigPoolSelector:
+    matchLabels: 
+      custom-kubelet: small-pods <2>
+  kubeletConfig: 
+    ImageMinimumGCAge:  <3>
```

Comment:
is it okay leaving it empty? Got the same question for 

ImageGCHighThresholdPercent: <4>
ImageGCLowThresholdPercent: <5>

Reply:
@weinliu I updated this example in another PR:   kubeletConfig:
    ImageMinimumGCAge: 0 <3>
    ImageGCHighThresholdPercent: 85 <4>
    ImageGCLowThresholdPercent: 80 <5>
```
  kubeletConfig:
    ImageMinimumGCAge: 0 <3>
    ImageGCHighThresholdPercent: 85 <4>
    ImageGCLowThresholdPercent: 80 <5>
```